projects/plugins/jetpack/_inc/lib/markdown/extra.php

```diff
@@ -2123,7 +2124,7 @@ function _hashHTMLBlocks_inHTML($text, $hash_method, $md_attr) {
 			#
 			$parts = preg_split($tag_re, $text, 2, PREG_SPLIT_DELIM_CAPTURE);
 
-			if (count($parts) < 3) {
+			if ((is_countable($parts) ? count($parts) : 0) < 3) {
```

Comment:
Here at least there's some point to the `is_countable()` check. But the code transform is still stupid.
```suggestion
			if ( ! $parts || count($parts) < 3 ) {
```